src/components/BounceRulesContainer/Modals/CreateRuleModal/index.jsx

```diff
@@ -136,7 +137,7 @@ const CreateRuleModal = ({
         <Column>
           <h1 className="h2.is-size-h1">Create a Bounce Rule</h1>
           {isInvalidInput && (
-            <Alert type="danger" onClick={handleModalClose} id="isInvalidInput">
+            <Alert type="danger" onClick={handleInvalidAlertClose}>
```

Comment:
Can you validate for invalid characters inline instead (show the error message below the text input) and reserve these alerts for server errors (like after submitting, if you get a server error, show that error in an alert banner)?

Reply:
👍

Reply:
Was wondering if the <Select> component has an isValid like the TextInputs